operator-common/src/main/java/io/strimzi/operator/common/AbstractOperator.java

```diff
@@ -496,38 +471,69 @@ public void accept(WatcherException e) {
     private void handleResult(Reconciliation reconciliation, AsyncResult<Void> result, Timer.Sample reconciliationTimerSample) {
         if (result.succeeded()) {
             updateResourceState(reconciliation, true, null);
-            successfulReconciliationsCounter.increment();
-            reconciliationTimerSample.stop(reconciliationsTimer);
+            getSuccessfulReconciliationsCounter(reconciliation.namespace()).increment();
+            reconciliationTimerSample.stop(getReconciliationsTimer(reconciliation.namespace()));
             LOGGER.infoCr(reconciliation, "reconciled");
         } else {
             Throwable cause = result.cause();
 
             if (cause instanceof InvalidConfigParameterException) {
                 updateResourceState(reconciliation, false, cause);
-                failedReconciliationsCounter.increment();
-                reconciliationTimerSample.stop(reconciliationsTimer);
+                getFailedReconciliationsCounter(reconciliation.namespace()).increment();
+                reconciliationTimerSample.stop(getReconciliationsTimer(reconciliation.namespace()));
                 LOGGER.warnCr(reconciliation, "Failed to reconcile {}", cause.getMessage());
             } else if (cause instanceof UnableToAcquireLockException) {
-                lockedReconciliationsCounter.increment();
+                getLockedReconciliationsCounter(reconciliation.namespace()).increment();
             } else  {
                 updateResourceState(reconciliation, false, cause);
-                failedReconciliationsCounter.increment();
-                reconciliationTimerSample.stop(reconciliationsTimer);
+                getFailedReconciliationsCounter(reconciliation.namespace()).increment();
+                reconciliationTimerSample.stop(getReconciliationsTimer(reconciliation.namespace()));
                 LOGGER.warnCr(reconciliation, "Failed to reconcile", cause);
             }
         }
     }
 
-    public Counter getPeriodicReconciliationsCounter() {
-        return periodicReconciliationsCounter;
+    @Override
+    public Counter getPeriodicReconciliationsCounter(String namespace) {
```

Comment:
My usual pet peeve about methods that return a value not needing to be called `get${Something}` when they can just be called `${something}`.